### ISSUE TITLE
collection export doesn't correctly handle quotes in song titles

### ISSUE THREAD

**Issue by bgentry** [matt-h/rdio-enhancer#65]
This is the first 3 lines of my exported Rdio collection:

```
Name,Artist,Album,Track Number
""Is This My Fate," He Asked Them","Story Of The Year","In The Wake Of Determination","12"
""The Take Over, The Breaks Over"","Fall Out Boy","Infinity On High (Deluxe Edition)","2"
```

Unfortunately, this isn't valid CSV. You can surround comma-containing strings in double quotes as you've done here, but then you also need to escape double quotes with a `""`. So I think the correct output would be:

```
Name,Artist,Album,Track Number
"""Is This My Fate,"" He Asked Them","Story Of The Year","In The Wake Of Determination","12"
"""The Take Over, The Breaks Over""","Fall Out Boy","Infinity On High (Deluxe Edition)","2"
```

I've confirmed with Golang's CSV parser that this is indeed parseable and valid.

**Comment by bgentry** [comment]
@matt-h awesome, thank you!

**Comment by gingold** [comment]
I ended up with the following:
```
Name,Artist,Album,Track Number
"""Is That Really It?"","Disjecta","Clean Pit & Lid","9"
```
Seems to be missing the final quote in the first field.

**Comment by matt-h** [comment]
Thanks for the feedback @gingold 
This should be fixed now in v2.10.3
